native/storybook/storybook.js

```diff
@@ -56,7 +56,7 @@ const styles = StyleSheet.create({
     flex: 1,
     flexDirection: 'column',
     justifyContent: 'center',
-    padding: spacingBase,
+    // padding: spacingBase,
```

Comment:
this will affect all the stories, is that what we want?

Reply:
Definitely not! I committed that by mistake 😅